mount/losetup_linux.go

```diff
@@ -94,6 +94,7 @@ func setupLoopDev(backingFile, loopDev string, param LoopParams) (*os.File, erro
 
 	// 2. Set FD
 	if _, _, err = ioctl(loop.Fd(), unix.LOOP_SET_FD, back.Fd()); err != nil {
```

Comment:
maybe we can use named return error and handle close fd in defer if error.

Reply:
This function should only close fd if it fails. need to return an unclosed fd so can't use `defer loop.Close()` 
https://github.com/containerd/containerd/blob/746cef0bc2afe8b8c4592fec3df966138bdcedff/mount/losetup_linux.go#L129-L136

Reply:
> maybe we can use named return error and handle close fd in defer if error.
> 
> Like:
> 
> ```go
> func setupLoopDev(backingFile, loopDev string, param LoopParams) (_ *os.File, retErr error) {
> 
>      ...
>      defer func() {
>           if retErr != nil {
>                  loop.Close()
>           }
>      }()
>      ...
> } 
> ```

Thanks, updated.